pages/check-out/index.js

```diff
@@ -0,0 +1,9 @@
+export default function Checkout({ selectedTable }) {
+  return (
+    <div>
+      <h2>Checkout Page</h2>
+      <p>Congratulations!</p>
+      <p>Selected Table: {selectedTable}</p>
```

Comment:
Where does ``{selectedTable}``come from? I can't see the table number on the checkout page.

Reply:
now you can find where it comes from